dev/tests/api-functional/testsuite/Magento/GraphQl/Customer/RequestPasswordResetEmailTest.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\GraphQl\Customer;
+
+use Magento\TestFramework\TestCase\GraphQlAbstract;
+
+class RequestPasswordResetEmailTest extends GraphQlAbstract
+{
+    /**
+     * @magentoApiDataFixture Magento/Customer/_files/customer.php
+     */
+    public function testCustomerAccountWithEmailAvailable()
+    {
+        $query =
+            <<<QUERY
+mutation {
+  requestPasswordResetEmail(email: "customer@example.com")
+}
+QUERY;
+        $response = $this->graphQlMutation($query);
+
+        self::assertArrayHasKey('requestPasswordResetEmail', $response);
+        self::assertTrue($response['requestPasswordResetEmail']);
+    }
+
+    /**
+     * @expectedException \Exception
+     * @expectedExceptionMessage
+     * GraphQL response contains errors: No such entity with email = customerNotAvalible@example.com, websiteId = 1
+     */
+    public function testCustomerAccountWithEmailNotAvailable()
+    {
+        $query =
+            <<<QUERY
+mutation {
+  requestPasswordResetEmail(email: "customerNotAvalible@example.com")
+}
+QUERY;
+        $this->graphQlMutation($query);
+    }
+
+    /**
+     * @expectedException \Exception
```

Comment:
Please, do not use exception annotations like `expectedException` and `expectedExceptionMessage`. This approach is deprecated in PHPUnit. Feel free to use the corresponding methods instead (`self::expectException`, `self::expectExceptionMessage`)

Reply:
done